addons/l10n_es/data/account_tax_data.xml

```diff
@@ -864,7 +874,16 @@
     <record id="account_tax_template_p_iva0_ns" model="account.tax.template">
         <field name="description">P_IVA0_NS</field>
         <field name="type_tax_use">purchase</field>
-        <field name="name">IVA Soportado no sujeto</field>
+        <field name="name">IVA Soportado no sujeto (Servicios)</field>
+        <field name="chart_template_id" ref="l10n_es.account_chart_template_common"/>
+        <field name="amount" eval="0"/>
+        <field name="amount_type">percent</field>
+        <field name="tax_group_id" ref="tax_group_iva_0"/>
```

Comment:
Except for the description, nothing is different.  I suppose they need to be reported, so should not we put different tag_ids then? (e.g. for the MOD303?)

Reply:
About the reports. For the standard reports like 303, it the same tag, but it's different for the SII.

Reply:
So we can just put the mod_303_61 tag for the moment?  Those things need an upgrade anyways.

Reply:
This not correct, this a purchase not a sale, the Key 61 for model 303 is only for sales. The purchase not subject don't appear in model 303.

https://www.agenciatributaria.es/AEAT.internet/en_gb/Inicio/Ayuda/Modelos__Procedimientos_y_Servicios/Ayuda_Modelo_303/Informacion_general/Instrucciones_modelo_303.shtml

Greetings